code_schemes/kalobeyei_s01e04.json

```diff
@@ -1,6 +1,6 @@
 {
-  "SchemeID": "Scheme-4a1c98e1",
-  "Name": "KAKUMA S01E04",
+  "SchemeID": "Scheme-fcfbd1f8",
+  "Name": "KALOBEYEI S01E04",
```

Comment:
Does e05 need to be renamed/re-id'd? The scheme is showing as renamed without changes for me, whereas all the other files have content changes.

Reply:
Thanks for catching this, have renamed the scheme name for e05 too.